test/test_time.cpp

```diff
@@ -14,56 +14,64 @@
 
 #include <gtest/gtest.h>
 
-#include <inttypes.h>
-
 #include <chrono>
+#include <cinttypes>
 #include <thread>
 
+#include "osrf_testing_tools_cpp/memory_tools/memory_tools.hpp"
+
 #include "rcutils/error_handling.h"
 #include "rcutils/time.h"
 
-#include "./memory_tools/memory_tools.hpp"
+using osrf_testing_tools_cpp::memory_tools::disable_monitoring_in_all_threads;
+using osrf_testing_tools_cpp::memory_tools::enable_monitoring_in_all_threads;
+using osrf_testing_tools_cpp::memory_tools::expect_no_calloc_begin;
+using osrf_testing_tools_cpp::memory_tools::expect_no_calloc_end;
+using osrf_testing_tools_cpp::memory_tools::expect_no_free_begin;
+using osrf_testing_tools_cpp::memory_tools::expect_no_free_end;
+using osrf_testing_tools_cpp::memory_tools::expect_no_malloc_begin;
+using osrf_testing_tools_cpp::memory_tools::expect_no_malloc_end;
+using osrf_testing_tools_cpp::memory_tools::expect_no_realloc_begin;
+using osrf_testing_tools_cpp::memory_tools::expect_no_realloc_end;
+using osrf_testing_tools_cpp::memory_tools::MemoryToolsService;
+using osrf_testing_tools_cpp::memory_tools::on_unexpected_calloc;
+using osrf_testing_tools_cpp::memory_tools::on_unexpected_free;
+using osrf_testing_tools_cpp::memory_tools::on_unexpected_malloc;
+using osrf_testing_tools_cpp::memory_tools::on_unexpected_realloc;
```

Comment:
it's a bit confusing to not see any usage of these in this file (I know they come from the `EXPECT_NO_MEMORY_OPERATIONS` macro).. I imagine there's a reason they either can't or shouldn't be embedded in the macro itself, but wanted to point out my concern (from a usability perspective) nonetheless

Reply:
I'll see if I can clean that up.